### ISSUE TITLE
(Suggestion) More operations on collections

### ISSUE THREAD

**Issue by lowoli** [Piotrekol/CollectionManager#40]
I like to categorize my beatmaps into different skill sets (e.g. aim, tapping, alt, etc.) and I wanted to generate a collection containing every beatmap I haven't categorized yet. I could do this by taking a collection with all my beatmaps and subtracting the merge of all my skill set collections, but the subtraction step doesn't seem to be doable (if it is, I'm not aware of how) without one of the two following solutions:

Subtraction: Take two collections, A and B, and create a new collection containing beatmaps in A which aren't in B
Negation: Create a new collection from A which contains every beatmap in the user's library that isn't in A
(Sorry if I over explained these, just want to be thorough)

Either of these two functions works for me personally, but I think the program would be better overall if it had both (and XOR too while we're on the topic).

TL;DR please add collection subtraction/negation and maybe even XOR

**Comment by Piotrekol** [comment]
Already possible in pre-release build - `Inverse` collection dropdown operation (does exactly what you mentioned for `Negation`).
You can even select multiple collections and perform inverse on all their maps at once.

**Comment by Deu07** [comment]
You can add "Subtraction", something like the inverse of "Combine Selected". This could work so that you select two collections and create a collection where maps A is not in collection B.

**Comment by lowoli** [comment]
Deuo07, using the new inverse function, there is a way to do subtraction. I'll write it here in case you were looking for a solution (or anyone else who stumbles upon this):
To find collection A minus B, you do the intersection of A and the inverse of B.

Would be cool to have a dedicated subtraction function, but this'll do for now.
